mender-convert-package

```diff
@@ -286,6 +286,10 @@ case "${MENDER_COMPRESS_DISK_IMAGE}" in
     log_info "Compressing ${img_path}.gz"
     run_and_log_cmd "pigz --best --force ${img_path}"
     ;;
+  zip)
+    log_info "Compressing ${img_path}.zip"
+    zip "${img_path}.zip" "${img_path}"
```

Comment:
What is the default compression level? Wondering if we should set `-9` explicitly.

Reply:
Default is `6`. 

I didn't see it explicitly set in the others either. 

Should it be added to all of them then?

Reply:
> I didn't see it explicitly set in the others either.

We do `pigz --best` and `pigz --best`

Reply:
:see_no_evil: